src/table-theme-colors.js

```diff
@@ -1,8 +1,23 @@
-import { isDarkColor } from './table/utils/color-utils';
+import { isDarkColor, isTransparentColor } from './table/utils/color-utils';
 
 export default function tableThemeColors(theme) {
-  const backgroundColor = theme.getStyle('object', 'straightTable', 'backgroundColor');
-  const isBackgroundDarkColor = isDarkColor(backgroundColor);
+  const qvInnerObject = document.querySelector('.qv-object .qv-inner-object');
+  const objectBackgroundColorFromCSS = qvInnerObject && window.getComputedStyle?.(qvInnerObject).backgroundColor;
```

Comment:
why wouldn't getComputedStyle exist?

Reply:
just in case that https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle#browser_compatibility

Reply:
yeah it is supported by everything, so should never be a problem

Reply:
sure, fixed